### DISCUSSION TITLE
<Title> The qualified invoice-based method (the Invoice System) for invoices and receipts

### DISCUSSION THREAD

**Discussion by NoriyukiKOIDE** [community/community#70603] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

In Japan, starting from this October, the Japanese government has implemented a strict management system called the The qualified invoice-based method (the Invoice System) for invoices and receipts. 
Invoices and receipts must be issued in compliance with this system, and those issued by GitHub are no exception. 
It appears that GitHub Inc. also completed registration as a qualified invoice issuer (T4700150079306), but the receipts we received here do not comply with the Invoice System. 
Could you please provide information on when and in what manner qualified invoices will be provided?

The qualified invoice-based method (the Invoice System)
https://www.nta.go.jp/taxes/shiraberu/zeimokubetsu/shohi/keigenzeiritsu/invoice_about.htm

**Comment by Sladeck** [comment]
Is there any update on that? Is there any setting in billing so that I can receive an invoice compliant with the law?

**Comment by NoriyukiKOIDE** [comment, marked as answer]
@Sladeck It looks the issue has alredy fiexed.
We can find the invoice issuer number on the recept since last December.

**Comment by VaskoReis** [comment]
Hello .. i try to find a place i cand download all my invoices/payments for 2024 ! can some one help me
